meson.build

```diff
@@ -423,7 +423,7 @@ if get_option('enable-wayland')
 You can disable the Wayland demo programs with -Denable-wayland=false.''')
     endif
 
-    wayland_scanner = find_program(wayland_scanner_dep.get_pkgconfig_variable('wayland_scanner'))
+    wayland_scanner = find_program(get_option('wayland-scanner'))
```

Comment:
doesn't this break the previous build as the option now **must** be specified for all builds? This should fallback to the pkgconfig variable since that's the most common use-case.

Reply:
> doesn't this break the previous build as the option now must be specified for all builds?

There is a default value, so you don't have to specify for all builds.  In native builds, if `wayland-scanner` is available it's likely in the `PATH`, without having to fiddle with the pkgconfig file.  BTW, with my pkgconfig file I couldn't find `wayland-scanner` anyway, so that looking into that file was quite useless

Reply:
Wayland upstream installs a `wayland-scanner.pc` which very definitely has the literal path to the pkg-config path. That works just fine.

There is more discussion on the issue [in weston!157](https://gitlab.freedesktop.org/wayland/weston/merge_requests/157).